crates/ghost_actor/src/ghost_actor.rs

```diff
@@ -263,97 +318,15 @@ pub trait GhostEndpoint<'lt, X: 'lt + Send + Sync, P: GhostProtocol> {
 
 /// Describes an actor that can be used within the "Ghost" actor system
 pub trait GhostActor<'lt, P: GhostProtocol, A: GhostActor<'lt, P, A>>: Send + Sync {
+    /// This is an indication that it is ok to begin processing
+    /// before actor_init is called, our self context may not be initialized
+    fn actor_init(&mut self) -> GhostResult<()> {
```

Comment:
`actor_init` sounds like a command.  I think this would be better named `is_initialized` or something like that?

Reply:
`can_init_actor()` and change the return type to bool

Reply:
updated comment